DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform-designer/Readme.md

```diff
@@ -0,0 +1,252 @@
+> ```
+
+5. Open platform designer from the Quartus GUI:
+
+   ![](Readme.md.assets/open-platform-designer-button.png)
+
+6. Add the following IPs to your system:
+   * Basic Functions > Bridges and Adaptors > Clock > Clock Bridge Intel FPGA IP
+   * Basic Functions > Bridges and Adaptors > Reset > Reset Bridge Intel FPGA IP
+   * Basic Functions > Bridges and Adaptors > Memory Mapped > JTAG to Avalon Master Bridge Intel FPGA IP
+
+7. Add the oneAPI IPs to your system and connect them as shown:
+
+   ![](Readme.md.assets/add-ip-platform-designer.png)
+
+   ![](Readme.md.assets/complete-system_platform-designer.png)
+
+   Don't forget to export the `IRQ` and `Exception Data Bus` signals, and update the base address to prevent overlap errors.
+
+8. Save the system by clicking `File > Save`, then click `Generate HDL...` and in the window that appears, click `Generate`.
+
+   ![](Readme.md.assets/generate-hdl.png)
+
+9. When generation is complete, close platform designer, then click the 'Start Compilation' button to compile the design.
+
+   ![](Readme.md.assets/start-compilation-quartus.png)
+
+### Additional Documentation
+- [Explore SYCL* Through Intel® FPGA Code Samples](https://software.intel.com/content/www/us/en/develop/articles/explore-dpcpp-through-intel-fpga-code-samples.html) helps you to navigate the samples and build your knowledge of FPGAs and SYCL.
+- [FPGA Optimization Guide for Intel® oneAPI Toolkits](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide) helps you understand how to target FPGAs using SYCL and Intel® oneAPI Toolkits.
+- [Intel® oneAPI Programming Guide](https://software.intel.com/en-us/oneapi-programming-guide) helps you understand target-independent, SYCL-compliant programming using Intel® oneAPI Toolkits.
+- [Intel® Quartus® Prime Pro Edition User Guide: Getting Started](https://www.intel.com/content/www/us/en/docs/programmable/683463/current/faq.html) introduces you to the Intel® Quartus® Prime Pro software.
+- [Intel® FPGA Software Installation and Licensing](https://www.intel.com/content/www/us/en/docs/programmable/683472/current/faq.html) describes how to license Intel® Quartus® Prime Pro software.
+- [Intel® Quartus® Prime Pro Edition User Guide: Platform Designer](https://www.intel.com/content/www/us/en/docs/programmable/683609/current/faq.html) describes the Intel® Platform Designer software.
+- [Intel® Quartus® Prime Pro Edition User Guide: Programmer](https://www.intel.com/content/www/us/en/docs/programmable/683039/current/programmer-user-guide.html) describes the Intel® Quartus® Prime Pro Programmer software.
+
+## Running the Sample
+
+Use the `test.bat` script in the `add-quartus/system_console` directory to flash the design to your development board, and launch the system console. The included `.tcl` scripts demonstrate how to use the System Console to interact with your IP through the JTAG Avalon Master IP on the FPGA.
```

Comment:
good question. these types of demos are often run on corporate laptops and use Windows. 

That script is simple enough that a Linux user could reproduce its functionality themselves, and since I have no convenient way of testing a linux version of test.bat, I do not want to include one.